packages/compiler/src/core/checker.ts

```diff
@@ -102,6 +102,14 @@ import {
   StdTypes,
   StringLiteral,
   StringLiteralNode,
+  StringTemplate,
```

Comment:
Do the names look good? Alternatives could have been
- `TemplateString`

Reply:
`StringTemplateLiteral` is probably more accurate